charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,63 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The Data Processing Service is a service that executes data processing pipelines containing analysis tasks performed on scientific data in the Data Coordination Platform (DCP). The outputs of these pipelines are used for scientific analysis and interpretation.  
+
+## Objectives
+
+The Data Processing Service develops the infrastructure to run pipelines reliably and at-scale for the DCP. The team implementing the service also operates the pipeline infrastructure for the Human Cell Atlas project. 
+
+## Definitions
+
+__pipeline:__ A collection of one or more functional tasks that operate on input data and, from that, make an output that is transformed input data, or derive features often used to interpret the input data. In a high throughput setting, these tasks are often automated to be performed in a batch setting.
+
+## In-scope
+
```

Comment:
This In-scope section is pretty light.  Sure, yes, Ops and Docs, but what about development, maintenance and improvement of the service itself?  What about the pipelines - which pipelines will the service run?  Only Standardized Pipelines?  Will we make a "best-effort" attempt to run anything that a third party comes up with?

Reply:
We had something about maintenance and improvement but the feedback we got was that such a thing was obviously in scope and need not be said. I am happy to add it back in if you feel it needs to be there. This charter will have to be updated once we better understand community pipelines.